### DISCUSSION TITLE
Cannot access the offer for microsoft azure

### DISCUSSION THREAD

**Discussion by SalmanYamin1995** [community/community#116888] [GitHub Education]
### Hi everyone,

Hi, I am trying to sign up for the microsoft azure offer. But after signing in with github on microsoft azure, it asks me for a verified university/school email. But i don't have that now, we use gmails in university.What can I do now?

**Comment by cxinu** [comment]
## Same issue

It is suppose confirm me as a student through GitHub, I don't have a student email either

![image](https://github.com/community/community/assets/72374438/d408a10d-e8cf-404c-8c78-813f3a4bf7d3)

**Comment by xylx521** [comment]
Now I'm having the same problem.

**Comment by y0geshx** [comment]
**Same issue**

How to resolve this issue?

**Reply by cxinu** [reply]
(Microsoft) GitHub says it's a (Microsoft) Azure issue, Which means they can't do anything about it, which means it's an over all skill issue for (Microsoft)

BTW I asked azure and they said if your student benefits are renewed they don't grant you those perks
